File: backend/ddang/src/main/java/com/ddang/ddang/notification/domain/NotificationType.java
package com.ddang.ddang.notification.domain;

import lombok.Getter;

@Getter
public enum NotificationType {

    MESSAGE("message"),
    BID("bid"),
    QUESTION("question"),
    ANSWER("answer");

    private final String value;

    NotificationType(final String value) {
        this.value = value;
    }
}

File: android/app/src/main/java/com/ddangddangddang/android/notification/NotificationType.kt
package com.ddangddangddang.android.notification

enum class NotificationType(private val value: String) {
    MESSAGE("message"), BID("bid");

    companion object {
        fun of(value: String): NotificationType? {
            return values().find { it.value == value }
        }
    }
}
